src/components/SignOut.tsx

```diff
@@ -23,7 +20,21 @@ export const SignOut: React.FC<SignOutProps> = ({
   }
   return (
     <>
-      <DialogDisclosure as={Button} color={color} store={dialog}>
+      <DialogDisclosure
+        store={dialog}
+        sx={{
+          color: color,
+          cursor: "pointer",
+          background: "none",
+          border: "none",
+          padding: 0,
+          font: "inherit",
+          textDecoration: "underline",
+          "&:hover": {
+            opacity: 0.8
+          }
+        }}
+      >
```

Comment:
Can no longer use this `as=` syntax here, so compensating for that in this change (similar elsewhere for other `DialogDisclosure` usage). Maybe should extract this as a separate component at some point.